src/app/components/MediaPlayer/README.md

```diff
@@ -0,0 +1,147 @@
+# Media Player
+
+## Description
+
+The `media-player` component exports two versions of our media player: an AMP version, and an Canonical version.
+
+At its core, this component returns an `iframe` that is designed to frame a media asset.
+The AMP variant will render an `amp-iframe` with a nested `amp-img` to use as a placeholder.
+The Canonical variant will render a placeholder, that when clicked will load the `iframe` into view.
+
+## When to use this component
+
+This component to be used at any point on the page, specifically when a media player is needed.
```

Comment:
```suggestion
This component is to be used at any point on the page, specifically when a media player is needed.
```
I know this is pre-existing, guessing it's meant to be either 'is to be used' or 'can be used'